docs/page-objects/rules.rst

```diff
@@ -1,782 +1,166 @@
-.. _rules-intro:
+.. _rules:
 
-Apply rules
-===========
+=====
+Rules
+=====
 
-Basic Usage
------------
+Rules are :class:`~.ApplyRule` objects that tell web-poet which :ref:`page
+object class <page-object-classes>` to use for a given combination of input URL
+and :ref:`item type <item-types>`.
```

Comment:
Technically, instead of item type one may be using just "instead_of" feature. Is it intentional that this is downplayed/removed from the updated docs?

Reply:
I keep forgetting that resolving a URL + item type is not a framework requirement.